clarity/src/vm/tests/simple_apply_eval.rs

```diff
@@ -1276,6 +1276,96 @@ fn test_stx_ops_errors() {
     }
 }
 
+#[test]
+fn test_bitwise() {
+    // NOTE: Type safety checks (e.g. that the 2nd argument to << and >> must be uint) are not included in this test.
+    // Tests for the type checker are included in analysis/type_checker/tests/mod.rs instead.
+
+    let tests = [
+        "(& 24 16)",            // 16
+        "(& u24 u16)",          // u16
+        "(^ 24 4)",             // 28
+        "(^ u24 u4)",           // u28
+        "(| 128 16)",           // 144
+        "(| u128 u16)",         // u144
+        "(~ 128)",              // -129
+        "(~ u128)",             // u340282366920938463463374607431768211327
+        "(~ u340282366920938463463374607431768211327)", // u128
+        "(>> u128 u2)",         // u32
+        "(<< u4 u2)",           // u16
+        "(& -128 -64)",         // -128
+        "(| -64 -32)",          // -32
+        "(^ -128 64)",          // -64
+        "(~ -128)",             // 127
+        "(>> -64 u1)",          // -32
+        "(<< -64 u1)",          // -128
+        "(>> 32 u2)",           // 8
+        "(<< 4 u4)",            // 64
+        "(| 1 2 4)",            // 7
+        "(| 64 -32 -16)",       // -16
+        "(| u2 u4 u32)",        // u38
+        "(& 28 24 -1)",         // 24
+        "(^ 1 2 4 -1)",         // -8
+        "(>> u123 u9999999999)", // u0
+        "(<< u123 u9999999999)", // u170141183460469231731687303715884105728
+        "(>> u240282366920938463463374607431768211327 u2402823)", // u1877205991569831745807614120560689150
+        "(<< u240282366920938463463374607431768211327 u2402823)", // u130729942995661611608235082407192018816
+        "(<< u340282366920938463463374607431768211455 u1)", // u340282366920938463463374607431768211454
+        "(<< -1 u7)",           // -128
+        "(<< -1 u128)",         // -1
+        "(>> -128 u7)",         // -1
+        "(>> -256 u1)",         // -128
+        "(>> 5 u2)",            // 1
+        "(>> -5 u2)",           // -2
+        "(<< 123 u9999999999)", // -170141183460469231731687303715884105728
+        "(>> 123 u9999999999)", // 0
```

Comment:
Can you also add `(<< -64 u121)` as a test vector?  Surprisingly, this will _panic_ in the Rust playground.  But, it works when I build this PR.